t/t-multiple-remotes.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env bash
+
+. "$(dirname "$0")/testlib.sh"
+
+reponame="$(basename "$0" ".sh")"
+
+prepare_mrtest(){
+  MRHOME=$TRASHDIR/home/mrhome
+  rm -rf $MRHOME
+  mkdir $MRHOME
+  HOME=$MRHOME
+  git config --global user.name "Git LFS Tests"
+  git config --global user.email "git-lfs@example.com"
+}
+
+prepare_consumer() {
+  local consumer=$1
+  mkdir $consumer
+  cd $consumer
+  git init
+  git remote add mr file://$smain
+  git remote add fr file://$sfork
+  git fetch mr
+  git fetch fr
+  git lfs install
+}
+
+prepare_remote() {
+  local repodir="$1"
+  mkdir -p "$repodir"
+  cd "$repodir"
+  git init --bare
+  git config http.receivepack false
+  git lfs install
+}
+
+prepare_forks () {
+  smain=$HOME/$reponame"-main-remote.git"
+  sfork=$HOME/$reponame"-fork-remote.git"
+  cmain=$HOME/$reponame"-main-repo"
+  cfork=$HOME/$reponame"-fork-repo"
+  rm -rf $smain
+  rm -rf $sfork
+  rm -rf $cmain
+  rm -rf $cfork
+  prepare_remote $smain
+  prepare_remote $sfork
+  prepare_consumer $cmain
+  git checkout -b master
+  git lfs track '*.bin'
+  git add --all
+  git commit -m "Initial commit"
+  git push -u mr master
+  git push -u fr master
+  #Add a .bin in main repo
+  touch a.bin
+  echo 1234 >> a.bin
```

Comment:
We typically prefer `printf` over `echo` in our codebase.  It's more consistent and flexible, and in some Windows environments, `echo` may insert CRLF, which makes our hashes change.

Reply:
I fixed it.
`printf "1234" > a.bin`